lib/utils/rabbitmq/src/lib/consumer.js

```diff
@@ -55,12 +55,15 @@ class Consumer {
       setupDeadLetterQueue(channel, conf.deadLetterQueues[1]),
       channel.prefetch(conf.mainQueue.prefetchLimit),
       channel.consume(conf.mainQueue.name, async(rabbitMessage) => {
-        const message = this.messageParser.toJSON(rabbitMessage);
+        let message = this.messageParser.toJSON(rabbitMessage);
         try {
           await messageHandler.handle(message);
           await channel.ack(rabbitMessage);
         } catch (err) {
           edebug('Failed to process message, returning to queue, error %o', err);
+          // TODO: remove this when there are no message in queue without metadata
```

Comment:
You could write a backlog item in jira instead of TODO

Reply:
https://jtrack.wdf.sap.corp/browse/CFA-8315